src/components/docs/MultiStepSnippet/styles.module.css

```diff
@@ -51,12 +59,29 @@
   padding: 0 20px;
 }
 
+.root li[role="tab"]:first-child > * {
+  border-top-left-radius: var(--snippet-border-radius);
+}
+
 @media small-tablet-breakpoint-max-query, 
 large-tablet-breakpoint-query and desktop-breakpoint-max-query {
   li[role="tab"]:before {
     content: '';
     padding-top: 100%;
   }
+
+  .root :global(.snippet-header) {
+    border-top-right-radius: 0;
+  }
+
+  .root div[role="tabpanel"], .root div[role="tabpanel"] > div > div {
```

Comment:
nit: 
1) newline separating selector
2) Is there any cleaner way to define this and the other `>` selectors? It's very inefficient. If not that's ok, but we should try to avoid expensive selectors: https://www.sitepoint.com/optimizing-css-id-selectors-and-other-myths/

Reply:
Unfortunately no. This is a big problem with styling on docusaurus. They have a limited amount of publicly exposed classes and the private classes are scoped with css modules